docs/requirements.txt

```diff
@@ -1,10 +1,10 @@
-certifi >=2022.9.24, <=2023.7.22
+certifi >=2024.07.04
 docutils<0.18
 six >=1.10, <=1.16.0
 python_dateutil >=2.5.3
-setuptools >=21.0.0, <=68.1.2
+setuptools >=70.0.0
 urllib3 >= 1.26.17
-paramiko >=2.11.0, <= 2.12.0
+paramiko >= 3.4.0
 pyjwt >=2.0.0, <=2.8.0
 requests >=2.20.1, <=2.31.0
```

Comment:
Let's also update this version.

Otherwise it looks ok.

Reply:
updated